src/site/_data/i18n/authors.yml

```diff
@@ -1,1116 +1,3347 @@
+    ja: Jeremy Colin
+    ko: Jeremy Colin
+    pt: Jeremy Colin
+    ru: Jeremy Colin
+    zh: Jeremy Colin
   description:
     en: Engineering Manager
+    es: Gerente de Ingeniería
+    ja: エンジニアリングマネージャー
+    ko: 엔지니어링 관리자
+    pt: Gerente de engenharia
+    ru: Менеджер по разработке
+    zh: 工程经理
 
 hbsong:
   title:
     en: Hongbo Song
+    es: Hongbo Song
+    ja: Hongbo Song
+    ko: Hongbo Song
+    pt: Hongbo Song
+    ru: Hongbo Song
+    zh: Hongbo Song
   description:
     en: Our latest news, updates, and stories by Hongbo Song.
+    es: Consulte las últimas noticias, actualizaciones e historias publicadas por Hongbo Song.
+    ja: Hongbo Song による最新のニュース、アップデート情報、記事などをご紹介します。
+    ko: Hongbo Song의 최신 뉴스, 업데이트 및 스토리.
+    pt: Nossas últimas notícias, atualizações e histórias por Hongbo Song.
+    ru: Последние новости и статьи от Hongbo Song.
+    zh: Hongbo Song 撰写我们的最新资讯、动态和故事。
 
 adamskret:
   title:
     en: Adam Skręt
+    es: Adam Skręt
+    ja: Adam Skręt
+    ko: Adam Skręt
+    pt: Adam Skręt
+    ru: Adam Skręt
+    zh: Adam Skręt
   description:
     en: Director of Digital & Performance Marketing, Orange Polska S.A.
+    es: Director de marketing digital y rendimiento de Orange Polska S.A.
+    ja: Orange Polska S.A. に所属するデジタル & パフォーマンスマーケティングディレクター
+    ko: 디지털 및 성능 마케팅 이사, Orange Polska SA
+    pt: Diretor de marketing digital e desempenho, Orange Polska S.A.
+    ru: Директор по цифровому и performance-маркетингу, Orange Polska S.A.
+    zh: Orange Polska S.A. 数字和绩效营销总监
 
 rafałfilipek:
   title:
     en: Rafał Filipek
+    es: Rafał Filipek
+    ja: Rafał Filipek
+    ko: Rafał Filipek
+    pt: Rafał Filipek
+    ru: Rafał Filipek
+    zh: Rafał Filipek
   description:
     en: Lead Frontend Developer, Orange Polska S.A.
+    es: Desarrollador principal de front-end en Orange Polska S.A.
+    ja: Orange Polska S.A. に所属するリードフロントエンド開発者
+    ko: 수석 프런트엔드 개발자, Orange Polska SA
+    pt: Desenvolvedor líder de front-end, Orange Polska S.A.
+    ru: Ведущий фронтенд-разработчик, Orange Polska S.A.
+    zh: Orange Polska S.A. 首席前端开发者
 
 marcinsierakowski:
   title:
     en: Marcin Sierakowski
+    es: Marcin Sierakowski
+    ja: Marcin Sierakowski
+    ko: Marcin Sierakowski
+    pt: Marcin Sierakowski
+    ru: Marcin Sierakowski
+    zh: Marcin Sierakowski
   description:
     en: Director of Sales Platforms Development & Maintenance, Orange Polska S.A.
+    es: Director de desarrollo y mantenimiento de plataformas de ventas en Orange Polska S.A.
+    ja: Orange Polska S.A. に所属する販売プラットフォーム開発 & メンテナンスディレクター
+    ko: 영업 플랫폼 개발 및 유지관리 이사, Orange Polska SA
+    pt: Diretor de desenvolvimento e manutenção de plataformas de vendas, Orange Polska S.A.
+    ru: Директор по развитию и поддержке торговых платформ, Orange Polska S.A.
+    zh: Orange Polska S.A. 销售平台开发与维护总监
 
 jiwoong:
   title:
     en: Jiwoong Lee
+    es: Jiwoong Lee
+    ja: Jiwoong Lee
+    ko: Jiwoong Lee
+    pt: Jiwoong Lee
+    ru: Чжи Ун Ли
```

Comment:
Most names don't seem to be translated but there are some cases like this, which are sort of oddly inserted in the set.

I presume @jimoong has suggested text for `ko`, which hasn't benn done.

Mostly I question the validity of, for nearly every author, just copying the English text to the other languages. I'd rather _not_ do this and fill in the gaps where we need to.

Reply:
ko: 이지웅

Reply:
@samthor 

> Most names don't seem to be translated but there are some cases like this, which are sort of oddly inserted in the set.
> 
> I presume @jimoong has suggested text for `ko`, which hasn't been done here.
> 
> Mostly I question the validity of, for nearly every author, just copying the English text to the other languages. I'd rather _not_ do this and fill in the gaps where we need to.

@samthor Thank you. Do you mean we should only leave the EN version (unless other versions are provided by the Authors themselves)? In this case, if the Description is translated, and the name is not translated (i.e. only the en: key is present - will the page fall back to English version (and display the Description in English?) 
We're not copying over the English version into other languages manually - this is how translations tools usually work: if something is not translated, the English version is used, but the key for the target language is still generated by the reverse parser in case there is no fallback to English. 
But we don't  mind post-editing manually and removing the unused keys. Let me know what you think. 
CC : @devnook